app/helpers/users_helper.rb

```diff
@@ -4,4 +4,8 @@ module UsersHelper
   def lead_school_user?
     defined?(current_user) && current_user&.lead_school?
   end
+
+  def can_view_drafts?
+    defined?(current_user).present? && current_user.present? && UserPolicy.new(current_user, nil).drafts?
```

Comment:
We need to include `defined?(current_user)` here for the component preview specs - `current_user` is undefined there (not just `nil`). Open to ideas about how to make this a bit less messy.

Reply:
I think `defined?(current_user)` is fine for now.